cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -266,15 +281,72 @@ func main() {
 		controllerCapabilities,
 	)
 
+	var capacityController *capacity.Controller
+	if (*capacityFeatures)[capacity.FeatureCentral] {
+		podName := os.Getenv("POD_NAME")
+		namespace := os.Getenv("POD_NAMESPACE")
+		if podName == "" || namespace == "" {
+			klog.Fatalf("need POD_NAMESPACE/POD_NAME env variables, have only POD_NAMESPACE=%q and POD_NAME=%q", namespace, podName)
+		}
+		controller, err := owner.Lookup(config, namespace, podName,
```

Comment:
While encouraged, we have not made it a requirement that a csi driver must run as a Kubernetes pod, and plenty of deployers have run CSI drivers and the sidecars as binaries, docker containers, and static pods (which do not show up in the api server).

Why must we have an owner? I see it as an optimization for cleaning up objects when you uninstall a driver. It doesn't really help the "run 2 of the same driver" since the scheduler can't distinguish.

We can look at adding a mode to support this in the future, but I would consider it a beta blocker. For now, let's make sure to document this requirement in the README.

Reply:
I've added a paragraph about this and a tracking issue: https://github.com/kubernetes-csi/external-provisioner/issues/465